source/index.js

```diff
@@ -57,7 +57,7 @@ module.exports = async (input = 'patch', options) => {
 	const pkgManager = options.yarn === true ? 'yarn' : 'npm';
 	const pkgManagerName = options.yarn === true ? 'Yarn' : 'npm';
 	const rootDir = pkgDir.sync();
-	const hasLockFile = fs.existsSync(path.resolve(rootDir, 'package-lock.json')) || fs.existsSync(path.resolve(rootDir, 'npm-shrinkwrap.json'));
+	const hasLockFile = fs.existsSync(path.resolve(rootDir, 'yarn.lock')) || fs.existsSync(path.resolve(rootDir, 'package-lock.json')) || fs.existsSync(path.resolve(rootDir, 'npm-shrinkwrap.json'));
```

Comment:
```suggestion
	const hasLockFile = fs.existsSync(path.resolve(rootDir, options.yarn ? 'yarn.lock' : 'package-lock.json')) || fs.existsSync(path.resolve(rootDir, 'npm-shrinkwrap.json'));
```

Since we're only interested in the lockfile that will be used by the configured package manager `np` is using.

Reply:
thanks @itaisteinherz